lib/config/validation.ts

```diff
@@ -297,10 +304,14 @@ export async function validateConfig(
             if (key === 'packageRules') {
               for (const [subIndex, packageRule] of val.entries()) {
                 if (is.object(packageRule)) {
-                  const resolvedRule = await resolveConfigPresets(
-                    packageRule as RenovateConfig,
-                    config
-                  );
+                  const resolvedRule = migrateConfig({
+                    packageRules: [
+                      await resolveConfigPresets(
+                        packageRule as RenovateConfig,
+                        config
+                      ),
+                    ],
+                  }).migratedConfig.packageRules[0];
```

Comment:
I don't understand this change? Why you nested it in a package rule?

Reply:
Check out the test to see the example that needs updating. It's a `group` extends within a package rule. Does work, but our validation is erroring it currently.

Reply:
ok, but looks pretty ugly